packages/lesswrong/server/editor/make_editable_callbacks.ts

```diff
@@ -447,7 +441,10 @@ function addEditableCallbacks<T extends DbObject>({collection, options = {}}: {
           updateType,
           commitMessage,
           changeMetrics,
-        });
+        }
+        // FIXME: See comment on the other Connectors.create call in this file.
```

Comment:
I might just copy the comment over, it's pretty annoying to be reading code and then have to interrupt the flow to read a comment somewhere else

Reply:
Idk, I'm not inclined to go against Jim here.